### DISCUSSION TITLE
coefficient of surface tension computation at boundary

### DISCUSSION THREAD

**Discussion by Rajatpareek** [InteractiveComputerGraphics/SPlisHSPlasH#184] [Q&A]
Hello,

I am doing some basic surface tension simulation. Where, three inclined plates (with different surface coating) having a water droplet on it. When this drop slide downward on plate it will make some _contact angle with plate_ (3 diff. angles for 3 plates).
So, for this case using surface tension model 4 can I give three different coefficient of surface tension computation at boundary **in a single simulation**?  So that physical phenomena of surface coating can be observed by measuring contact angles.

Thank You

**Comment by janbender** [comment]
You can define multiple phases to solve this. So you could define 4 fluids with different IDs and different material properties.
